templates/create-daml-app/ui/src/components/UserList.tsx.template

```diff
@@ -29,7 +30,7 @@ const UserList: React.FC<Props> = ({users, onFollow}) => {
                 className='test-select-add-user-icon'
                 onClick={() => onFollow(user.username)} />
             </List.Content>
-            <List.Header className='test-select-user-in-network'>{user.username}</List.Header>
+            <List.Header className='test-select-user-in-network'>{partyToAlias.get(user.username) ?? user.username}</List.Header>
```

Comment:
This will fail if a user has not created an Alias contract. We make sure that all test users have an associated Alias in the setup script. But if someone allocated a party setup script after the setup (say via daml ledger), this would fail. I'd prefer the fallback to party names instead of a hard failure.